src/main/java/uk/gov/hmcts/darts/common/entity/Courthouse.java

```diff
@@ -24,31 +22,17 @@ public class Courthouse {
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Integer id;
 
-    @Column(name = "r_courthouse_object_id", length = 16)
-    private String legacyObjectId;
+    @Column(name = "code")
+    private short code;
 
-    @Column(name = "c_code", length = 32)
-    private String code;
+    @Column(name = "courthouse_name")
+    private String courthouseName;
 
-    @Column(name = "c_id", length = 32)
-    private String idString;
+    @CreationTimestamp
+    @Column(name = "created_date_time")
+    private Timestamp createdDateTime;
```

Comment:
Please can we update the Timestamp types here to be `java.time.OffsetDateTime` instead in all places

Reply:
What is the import package for timestamp? I can't find it.

Reply:
```suggestion
    private OffsetDateTime createdDateTime;
```
Change import. Remove line 13 `import java.sql.Timestamp;`

Reply:
ACCEPT